src/article/article.ts

```diff
@@ -70,3 +73,31 @@ const getHeadings = (articleDom: DocumentFragment): Heading[] => {
     return headings;
   }, new Array<Heading>());
 }
+
+const getHeader = (articleDom: DocumentFragment): string => {
```

Comment:
are there tests for this function?

Reply:
Nope - it was a quick experiment. I've started gathering a TODO list above for thing that would need to be done to make this mergeable

Reply:
Pushed up a new test case that includes extending `validArticleHtml` with more example article data from encoda. It's not pretty though, and we could likely get away with much simpler, though less "real", HTML for the purposes of testing the behaviour - thoughts?